expressEngine/product/service/product.js

```diff
@@ -2,34 +2,29 @@
 import { BaseService, DBService } from '../../lib/service';
 
 // import collections
-import { Product, ProductDetail } from '../../model';
+import { Product } from '../../model';
 
 // import messages
-import {success} from '../../cms/product';
-
-//import validateRequired
-import { validateRequired } from '../../lib/validationHandler';
+import { success } from '../../cms/product';
 
 class Service extends BaseService {
   registerProduct = async (data) => {
-    try {
-      const { name, supplierId, unitPrice, productPackage, isDiscontinued } = data,
-      requiredFields = ['name', 'supplierId','unitPrice'];
-
-      validateRequired(data, requiredFields);
-      const product = await DBService.create(Product, {
-        name,
-        supplierId,
-        unitPrice,
-        package: productPackage,
-        isDiscontinued
-      });
+    const {
+      name, supplierId, unitPrice, productPackage, isDiscontinued,
```

Comment:
If we provide supplierId as 1 then product is created. I think this should not have happened. instead of that supplierId should only take supplier's _id

Reply:
@ayushj117 please create a new issue for this bug.